performance-timeline/case-sensitivity.any.js

```diff
@@ -27,3 +27,37 @@
     assert_equals(mixedList.length, 0, "getEntriesByName('" + location2 + "').length");
 
   }, "getEntriesByName values are case sensitive");
+
+  async_test(function (t) {
+    // Test type/buffered case sensitivity.
+    observer = new PerformanceObserver(
+      t.step_func(function (entryList, obs) {
+        assert_unreached("Observer should not be called.");
```

Comment:
nit: let's use a slightly different error message for the assert_unreached so that a developer can know which one fails if that happens.